pallets/cash/src/params.rs

```diff
@@ -19,3 +19,10 @@ pub const TRANSFER_FEE: Quantity = Quantity::from_nominal("0.01", CASH);
 
 /// Number of blocks between HTTP requests from offchain workers to open oracle price feed.
 pub const ORACLE_POLL_INTERVAL_BLOCKS: u32 = 10;
+
+/// The number of blocks to wait before switching sessions
+#[cfg(debug_assertions)]
+pub const SESSION_PERIOD: u32 = 2;
```

Comment:
just noticed we set this to 50 before, makes integration tests go slower, what was the reason behindthis?

Reply:
can be 2 in test, but running live it is really spammy to be changing sessions every 2 blocks